src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double firstSide;
+    private final double secondSide;
+    private final double height;
+
+    public IsoscelesTrapezoid(double longerBase, double shorterBase, double height, String color) {
+        super(color);
+        this.firstSide = longerBase;
+        this.secondSide = shorterBase;
+        this.height = height;
+    }
```

Comment:
The parameter names in the constructor are not matching with the class variables. Consider changing the parameter names to 'firstSide', 'secondSide' and 'height'.

Reply:
done